utils/amt-integration-tests/.vscode/settings.json

```diff
@@ -0,0 +1,110 @@
+{
+    "python.analysis.extraPaths": ["./change_to_module_name", "./tests"],
```

Comment:
We should change this `"./change_to_module_name"`

Reply:
Done